bears/general/LicenseCheckBear.py

```diff
@@ -0,0 +1,40 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.DistributionRequirement import (
+    DistributionRequirement)
+
+
+@linter(executable='licensecheck',
+        output_format='regex',
+        output_regex=r'.*: .*UNKNOWN',
+        result_message='No license found.')
+class LicenseCheckBear:
+    """
+    Attempts to check the given file for a license, by searching the start
+    of the file for text belonging to various licenses.
+
+    For Ubuntu/Debian users, the `licensecheck_lines` option has to be used in
+    accordance with the `licensecheck_tail` option.
```

Comment:
we use double backticks, and I'm not sure if RST handles single ones properly. Could you change that everywhere? :)